js/main.js

```diff
@@ -108,6 +111,5 @@ function renderData() {
   $dayLength.textContent = astronomyData.dayLength + ' HRS.';
   var $moonDistance = document.querySelector('.moon-distance');
   $moonDistance.textContent = astronomyData.moon_distance + ' Miles';
+  window.addEventListener('DOMContentLoaded', renderData);
```

Comment:
This will never run, because you are certainly adding this event listener after the DOMContent has already loaded. If you remove this event listener entirely, what happens?